Doc/library/sys.rst

```diff
@@ -926,7 +926,8 @@ always available.
 
     A list of :term:`meta path finder` objects that have their
     :meth:`~importlib.abc.MetaPathFinder.find_spec` methods called to see if one
-    of the objects can find the module to be imported. The
+    of the objects can find the module to be imported. by default, holds entries
```

Comment:
This is not grammatically correct: a sentence should start with a capital letter and contain a verb.

Reply:
Thank you, how about
"By default, it holds entries to handle the standard kinds of modules (.py files, extension modules...)."